scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -47,17 +47,17 @@
   [node-name->resource-map resource-keyword]
   (->> node-name->resource-map vals (map resource-keyword) (filter some?) (reduce +)))
 
-(defn total-gpu-resource
+(defn total-text->scalar-resource
```

Comment:
should this be called resource-count-map ?
or at least get-text->scalar-resource-total 

seems like "total-text" is converted to "scalar-resource"

Reply:
hm, reasoning for this is because the parallel for scalar resources is `total-scalar-resource`, and disk/gpu resources are represented with type `text->scalar` instead. however, I do agree that this current name looks like "total-text" -> "scalar-resource". @dposada what do you think?

Reply:
how about `total-map-resource`?

Reply:
sounds good to me